ducktape/tests/logger.py

```diff
@@ -38,3 +38,12 @@ def configured(self):
 
     def configure_logger(self):
         raise NotImplementedError("configure_logger property must be implemented by a subclass")
+
+    @staticmethod
+    def close_logger(logger):
```

Comment:
Why is this a static instead of just a member variable? Just to avoid the `if logger is not None` at the call site?